apps/datahub/src/app/search/search-header/search-header.component.html

```diff
@@ -16,6 +16,7 @@ <h1 class="flex mel-page-title md:mb-12" translate="">
     </div>
     <div class="h-64" gnUiSearchStateContainer="headerSearch">
       <mel-datahub-results-list-carousel
+        [records]="records$ | async"
```

Comment:
I'm wondering if you could just pass the filter values from the `mainSearch` search state down to the `results-list.comonent` (which has its own `headerSearch` search state) here and then make the `results-list.component` listen to the producer changes.